### DISCUSSION TITLE
Profile Photo Not Showing Up After Commits

### DISCUSSION THREAD

**Discussion by tclare** [community/community#11784] [Other Feature Feedback, Questions, & Ideas]
Hey team! 👋
Firstly want to say I love using Github: seamless UI and overall fabulous UX. One thing: sometimes after commits, my avatar doesn't show on the Latest Commit card. Thought you may like to know! Attaching photo for visual proof.
<img width="1674" alt="Screen Shot 2022-02-19 at 2 11 38 PM" src="https://user-images.githubusercontent.com/38092715/154815539-c4715eca-d9d4-42d2-a97b-37ae2405bdac.png">

**Comment by capncrockett** [comment]
Same issue here. Tried refreshing the page. Same deal. Opening the image in a new tab gave me:
"""
Error 503 Backend is unhealthy
Backend is unhealthy

Guru Mediation:
Details: cache-pdx12322-PDX 1645545467 866540388

Varnish cache server
"""
![Screenshot from 2022-02-22 07-57-58](https://user-images.githubusercontent.com/71466450/155170195-bedf711e-908c-4c29-8ed0-f290e68322e3.png)

**Reply by LoginAGo** [reply]
I've recently been getting this problem. I was trying to see if my style.css file would load on my website https://oneenterprisestech.github.io, and it wouldn't load, so I tried to view it as raw to see if the code would load that way. Got the same errors. Also, my website is live for SEO. This could mess up reviews (that is something I am worried about)... Do you think it is a GitHub issue, like the server has an error or something?

![image](https://user-images.githubusercontent.com/26009667/155181554-06b43aa1-8df0-47f0-ae5f-9b2ed39c4eab.png)

**Reply by LoginAGo** [reply]
Also, errors with loading images vital to my website occur. 
![image](https://user-images.githubusercontent.com/26009667/155182161-3dab12ef-5569-45ef-8d07-a111acff7c33.png)

**Reply by LoginAGo** [reply]
Now it's not loading the banner (header) image and several other images.
![image](https://user-images.githubusercontent.com/26009667/155182590-b3e649f1-1626-4c7b-8c31-ed8808c15a32.png)

**Reply by LoginAGo** [reply]
Ugh, it just switches through all of the files. I don't get it.

**Reply by skrishnan2001** [reply]
Did you find a solution to this problem? I'm facing the same issue!

**Comment by aleebaba3594** [comment]
mark my reply i want get badge
